backend/src/routes/api/namespaces/namespaceUtils.ts

```diff
@@ -32,6 +27,34 @@ const checkNamespacePermission = (
   });
 };
 
+const checkNamespacePermission = (
```

Comment:
Just a nit, you can call this `checkAdminNamespacePermission` or something if you want.